### PR TITLE
docs(README): clarify defaults (`pluginOpts`)

### PR DESCRIPTION
**Pull request by davestaab**
It's not clear if these are the defaults or if they're examples of how the user could use something different. 
If they are just examples what are the defaults? (maybe undefined?)
I've made several pluginOpts more consistent in formatting and punctuation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,10 +47,10 @@ module.exports = {
 
 + `test`: JS file extension regex. Default: `/\.js($|\?)/i`
 + `comments`: Preserve Comments. Default: `/^\**!|@preserve|@license|@cc_on/`, falsy value to remove all comments. Accepts function, object with property test (regex), and values.
-+ `sourceMap`: Default: uses [webpackConfig.devtool](https://webpack.js.org/configuration/devtool/). Set this to override that.
++ `sourceMap`: Configure a sourcemap style. Default: [webpackConfig.devtool](https://webpack.js.org/configuration/devtool/)
```
Comment (michael-ciniawsky) [inline README.md]: `Configure source maps`

File: README.md [hunk 2]
```diff
@@ -47,10 +47,10 @@ module.exports = {
...
 + `parserOpts`: Configure babel with special parser options.
-+ `babel`: Pass in a custom babel-core instead. `require("babel-core")`
-+ `minifyPreset`: Pass in a custom minify preset instead - `require("babel-preset-minify")`.
++ `babel`: Pass in a custom babel-core instead. Default: `require("babel-core")`
```
Comment (michael-ciniawsky) [inline README.md]: babel-core => ``` `babel-core` ```

File: README.md [hunk 3]
```diff
@@ -47,10 +47,10 @@ module.exports = {
...
++ `minifyPreset`: Pass in a custom minify preset instead. Default: `require("babel-preset-minify")`
```
Comment (michael-ciniawsky) [inline README.md]: minify => ``` `babel-minify` ```

**Comment by davestaab** [comment]
So need me to do anything? 

Sorry, I'm not sure if this is waiting on me. 

Just let me know.

**Comment by michael-ciniawsky** [comment]
Address the minor style nitpicks of the review please, otherwise it's g2g :)

**Comment by davestaab** [comment]
Alrighty, so I think I've addressed the changes. Let me know if I've missed something. 

Also, I accidentally made that change with a different email address. The CLA bot picked up that name and says I need to sign the CLA again. But when I go to the link I've already signed it. 

I amended the commit and force pushed it to my branch. (not sure if that was the right thing to do)
The commit is now good but the CLA bot still thinks my other email made made a commit. 

Sorry, I'm new to this and learning. Thanks so much for the help!

**Comment by michael-ciniawsky** [comment]
1. The e-mail address of your github account must match the e-mail address in your `global` `.gitconfig` (`$HOME/.gitconfig`), every commit is signed with the author  (`name <email>`)

```
$ git log
```

```
commit 00e20a4e516218fb04cdd9ce67df44aea5ae7003
Author: Michael Ciniawsky <michael.ciniawsky@gmail.com>
Date: Fri Mar 9 17:30:36 2018 +0100
````

2. Try to `git rebase`  your commits and squash them into one commit, which should be signed with the correct (CLA signed) e-mail

```bash
git rebase -i HEAD~3
```

```
pick 3854898  <commit message>  (correct author)
squash 00e20a4 <commit message> (wrong author)
squash fe5fedb <commit message> (wrong autor)
```

>⚠️ Don't squash your changes into an existing commit **before** your `git checkout -b my-branch` (a commit you didin't create yourself)

```bash
$ git push my-fork my-branch --force
```

**Comment by davestaab** [comment]
Thanks for the reply. 

I did something similar to what you suggested. Instead of a squash, I amended the commit with the bad email. All my commits now have a github (CLA signed) email. (I actually did this yesterday before my last reply)

It appears the CLA bot is happy now and says "All committers have signed the CLA."

When I check the commits it shows both have passed the CLA check.

Let me know if there's anything else I need to do. Sorry this has been so much work. =D

**Pull request merged**